packages/fx-core/src/common/featureFlags.ts

```diff
@@ -13,6 +13,20 @@ export function isFeatureFlagEnabled(featureFlagName: string, defaultValue = fal
   }
 }
 
+/**
+ * Update feature flags related to GA.
+ */
+export function initializeGAFeatureFlags(): void {
+  if (isFeatureFlagEnabled(FeatureFlagName.GAPreview, false)) {
```

Comment:
What if user changed one of the feature flags after fxcore initialized?

Reply:
If user changes configuration in VSC, they are instructed to reload window to take effect. This function is exported and called by VSC and CLI during activation.